### PR TITLE
Adding Script to run SwiftFormat and SwiftLint and custom rules config files

### LINKED ISSUE #8: Add SwiftLint and SwiftFormat to the project.
## Description
Add SwiftLint and SwiftFormat config files into directory with package. 
Add script to run SwiftFormat and SwiftLint
_If possible, add SwiftLint violations check for build phase_

## DoD

See the files in the [repo](https://github.com/ios-course/ironfoudation-team-project):
- `.swiftformat`
- `.swift-version`
- `.swiftlint.yml`

### PR DESCRIPTION
**Pull request by iBYarchitect**
### Summary
This PR resolve #8.

### Script using
To use SwiftLint and SwiftFormat, follow these steps:
1. Open the terminal.
2. Navigate to the root directory of SimpleNetworkService package .
3. Run `SwiftLint; Swiftformat .`

### Changes
- .swiftformat
- .swift-version
- .swiftlint.yml

### Checks to complete

- [x] The PR title has a conventional commit style (short and descriptive).
- [x] The PR description includes essential information that helps understand the purpose of the changes.
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] My code follows our style guide.
- [x] I've performed a self-review of my code.
- [x] Every non-private interface is documented properly.
- [x] I've added tests for my code (if it makes sense).
- [x] I've updated `CHANGELOG.md` file.
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

**Review by kirpichonok — CHANGES_REQUESTED** [review]
Almost done, but there are several violations left in the `HTTPAPIEndpoint` and `NetworkService` files.  Please, correct them to bring the project into compliance with our rules completely.
I'm not sure that we really need this script. It's not difficult to run swiftlint and swiftformat commands manually and not to add 23 lines of code. That is more robust, I think.
Screenshot 2023-09-01 at 12 52 38

File: .RunSwiftFormatSwiftLint.sh
Comment (kirpichonok) [inline .RunSwiftFormatSwiftLint.sh:1]: Will be great, if you remove the script.

Comment (iBYarchitect) [reply .RunSwiftFormatSwiftLint.sh:1]: Deleted with 9b39b8a

**Review by kirpichonok — APPROVED** [review]
LGTM

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM to unblock

**Pull request merged**